mappings/net/minecraft/util/collection/PackedIntegerArray.mapping

```diff
@@ -3,6 +3,21 @@ CLASS net/minecraft/class_3508 net/minecraft/util/collection/PackedIntegerArray
 	FIELD field_15632 size I
 	FIELD field_15633 elementBits I
 	FIELD field_15634 maxValue J
+	FIELD field_24078 INDEX_PARAMETERS [I
+		COMMENT Magic constants for faster integer division by a constant.
+		COMMENT
+		COMMENT This is computed as (n * scale + offset) >> (32 + shift). For a divisor n,
```

Comment:
```suggestion
		COMMENT <p>This is computed as {@code (n * scale + offset) >> (32 + shift)}. For a divisor n,
```
Otherwise this kills html and javadoc